### ISSUE TITLE
Add a new trigger for automated rules

### ISSUE THREAD

**Issue by Gsyltc** [firefly-iii/firefly-iii#6760]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

Some automatic transactions may be conditional on an account balance (positive balance, reaching a maximum balance).

Ex: 
Revolut allows you to put a certain amount (e.g. 5€ per day) in a vault.
This payment is conditional. the balance of the original account need to be higher than the amount to be deposited in the safe. Otherwise the transaction is not done.

### Solution

Add 2 triggers:

 - Account balance is greater than 
 - Account balance is less than

### What are alternatives?

No alternatives

Thanks

**Comment by JC5** [comment]
That's not a bad idea, but these triggers fire when transactions are created or updated. In other words, by then you're already too late to respond, the transaction is created already. Since a rule can't create other transactions, you may be better off with webhooks?

**Comment by Gsyltc** [comment]
Yes, but I see an action : "Delete opération" 
It's not the most elegant but it could do the trick
It is certain that it would be better to have an audit before placing the order in the accounts

**Comment by JC5** [comment]
That's true, and the balance only changes based on transactions anyway. I'll put it on the list!
